src/erlfmt_cli.erl

```diff
@@ -311,6 +323,28 @@ parse_opts([delete_pragma | _Rest], _Files, _Exclude, #config{pragma = require})
     {error, "Cannot use both --require-pragma and --delete-pragma options together."};
 parse_opts([delete_pragma | Rest], Files, Exclude, Config) ->
     parse_opts(Rest, Files, Exclude, Config#config{pragma = delete});
+parse_opts([{range, String} | Rest], Files, Exclude, Config) ->
+    % Ad-hoc parsing. Mitigation for the absence of "couple of integers" direct support.
+    Range1 = string:split(String, ",", all),
+    Range2 = [list_to_integer(X) || X <- Range1],
+    Range3 =
+        case Range2 of
+            [X] ->
```

Comment:
I think is okay to not support a single line, except if this is required by the language server

Reply:
Well, supportting that just take this 2 simple lines. Does the right thing TM + 'Be liberal in what you accept' principle :)

Reply:
I guess I would say being literal is that a range only supports a range, not a single line, but that is just my opinion.  Not going to block based on this.